src/board.scss

```diff
@@ -1,24 +1,29 @@
 #board {
+  --col-count: 10;
+  --row-count: 10;
+  --width: 150px;
+  width: calc((var(--col-count) + 2) * var(--width));
```

Comment:
Is this a `sass` feature that I've never used?

Reply:
It's using https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables.  I tried `sass` variables, but they didn't seem to work.  I didn't look into it.  I figured we can always `s/--/$/g` (or whatever) eventually if we want to leverage `sass` variables.